examples/curator-s3-deployment/patches/elasticsearch.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+- op: add
+  path: /spec/template/spec/initContainers/-
+  value:
+    name: install-plugins
+    command:
+      - ciao
```

Comment:
Is this intended? 😄 👋

Reply:
of course was not intended 👯  i remove it NOW